lib/tapioca/runtime/reflection.rb

```diff
@@ -132,7 +132,9 @@ def qualified_name_of(constant)
       sig { params(method: T.any(UnboundMethod, Method)).returns(T.untyped) }
       def signature_of(method)
         T::Utils.signature_for_method(method)
-      rescue LoadError, StandardError
+      rescue LoadError, StandardError => e
+        $stderr.puts "Error loading signature for method: #{method.owner}##{method.name}: #{e.inspect.squish}"
+        $stderr.puts "  The generated signature is likely inaccurate..."
```

Comment:
I agree that this is not the ideal place to display errors. In general, `signature_of` could be called to do some other processing, so the message displayed which says "generated signature is likely inaccurate" might not even be correct if such a use-case were to fail.

I think we can add an optional block to this method that would be called with the raised `LoadError` or `StandardError` instance, if supplied. That would allow us to handle the error in gem RBI generation pipeline and maybe skip it in other locations.

So something like would be great:
```ruby
def signature_of(method, &blk)
  # ...
rescue LoadError, StandardError => e
  blk.call(e) if blk
end
```

Reply:
I definitely agree with the general approach in allowing the caller to react accordingly, but feel relying on the unnamed/implicit block for the method is a bit unintuitive. I think I might try implementing the `error_handler: -> {...}` from the `Tapioca::Dsl::Pipeline`  on to the `Tapioca::Gem::Pipeline` just to see how it feels.